modules/drivers/usb/api/rtos_usb.h

```diff
@@ -250,6 +250,9 @@ static inline void rtos_usb_endpoint_stall_clear(rtos_usb_t *ctx,
     XUD_ClearStallByAddr(endpoint_addr);
 }
 
+void rtos_usb_enter_test_mode(rtos_usb_t *ctx,
```

Comment:
Please add documentation for this function.

Reply:
Done